### PR TITLE
Add daniel's favorite number

### PR DESCRIPTION
**Pull request by dhong98**

### CODE REVIEW

File: numbers.csv [hunk 1]
```diff
@@ -4,3 +4,4 @@ Sue,42
 Diana,27
 Jason,9
 Sophia, 10
+Daniel, 2
```
Comment (sfelshin) [inline numbers.csv:7]: Add missing newline at end of file.  Some unix programs can only read a line if it ends with a newline, and if the last line has no trailing newline, they'll ignore that line.

**Review by sfelshin — CHANGES_REQUESTED** [review]
It looks like you need to resolve some merge conflicts.  If you need help, ask David or me.

File: numbers.csv (hunk 1 above)
Comment (alvaromorales) [reply numbers.csv:7]: Does emacs have a setting where newlines at the end of files are added automatically? Other text editors handle this by default.

**Comment by sfelshin** [comment]
> Does emacs have a setting where newlines at the end of files are added automatically? Other text editors handle this by default.

Good question!  It never occurred to me.  Here's the answer: https://www.gnu.org/software/emacs/manual/html_node/emacs/Customize-Save.html

The default value of `require-final-newline` is `nil`.  I can either:
   1. Change it to `t` in the [shared emacs init file](git:infolab-core/common-files/emacsrc).  Pro: Convenient for pretty much everyone. Con: Annoying if anyone _doesn't_ want it; they'd have to turn it off _below_ where they load the `common-emacs` file.
   1. Recommend in [Intro: `.emacs`](https://projects.csail.mit.edu/cgi-bin/wiki/view/Infolab/Intro#A_61.emacs_61) that people set it to `t`.

I lean toward the first alternative.  @alvaromorales, do you have an opinion?

**Comment by alvaromorales** [comment]
I support changing it in the shared emacs config!

**Comment by sfelshin** [comment]
@dhong98, one other comment, which is that when you created the PR, you should have assigned me as the reviewer.  (Eventually, you and your fellow UROPs will review each other's commits, but for now I'll review them.)

**Comment by sfelshin** [comment]
> I support changing it in the shared emacs config!

See https://github.com/infolab-csail/infolab-core/pull/70.

**Pull request closed without merging**